.github/workflows/merge-prometheus.yaml

```diff
@@ -19,6 +19,10 @@ jobs:
       upstream: prometheus/prometheus
       downstream: openshift/prometheus
       sandbox: rhobs/prometheus
+      # .gitignore was adjusted in https://github.com/openshift/prometheus/pull/227/commits/1303b9a0eac1ed3fd24494457c8344d45d35e871
+      # to keep track of the generated UI assets.    
+      restore-downstream: >-
+        .gitignore
```

Comment:
Could you provide some context on why we are including gzipped UI assets only? I could be missing something but I'm assuming cluster admins will still want to use the built-in UI, which would imply that these are being decompressed somewhere?

Reply:
No we want the zipped assets in our binary. If the uncompressed assets are present, they end up in the binary causing excessive bloat.
Assets are served compressed and decompressed by the requesting http client.